LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Steven Murray
```

Comment:
I'm not super familiar with how software copyright works. Should this be copyrighted by "the authors"? Or our group? I know things like pyuvdata are copyright "The Radio Astronomy Software Group", but not sure how that plays out in practice. (I'm also not sure if I'm making a big deal out of nothing, so feel free to ignore this...)